mp/src/game/client/momentum/ui/SettingsPanel/HUDSettingsPanel.cpp

```diff
@@ -158,17 +160,25 @@ HUDSettingsPanel::HUDSettingsPanel(Panel *pParent, Button *pAssociate) : BaseCla
 void HUDSettingsPanel::OnPageShow()
 {
     BaseClass::OnPageShow();
+    
+    m_pSpeedometerGameType->ActivateItemByRow(g_pGameModeSystem->GetGameMode()->GetType());
+    // loads fields
+    OnTextChanged(m_pSpeedometerGameType, "");
```

Comment:
Does not calling `SilentActivate` for the combobox not already do this?

Reply:
It was not. Looking into it it was because the [`OnTextChanged()` override for the `CvarComboBox`](https://github.com/momentum-mod/game/blob/4674e718631c8dffaa7baae1171fd4cde43120c4/mp/src/vgui2/vgui_controls/CvarComboBox.cpp#L101) doesn't post the action signal unless the value has changed. 

Because the gametype combobox's value is set to unknown in the constructor, the action signal is not posted when first opening to the unknown gamemode settings. Removing the setting in the constructor allowed it to work properly.